dlf/ext_tables.php

```diff
@@ -217,9 +217,108 @@
 	)
 );
 
+$TCA['tx_dlf_basket'] = array (
+	'ctrl' => array (
+		'title'     => 'LLL:EXT:dlf/locallang.xml:tx_dlf_basket',
+		'label'     => 'label',
+		'tstamp'    => 'tstamp',
+		'fe_user_id' => 'user_id',
+		'languageField' => 'sys_language_uid',
+		'transOrigPointerField' => 'l18n_parent',
+		'transOrigDiffSourceField' => 'l18n_diffsource',
+		'default_sortby' => 'ORDER BY label',
+		'delete'	=> 'deleted',
+		'dynamicConfigFile' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath($_EXTKEY).'tca.php',
+		'iconfile'	=> \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath($_EXTKEY).'res/icons/txdlflibraries.png',
+		'rootLevel'	=> 0,
+		'dividers2tabs' => 2,
+		'searchFields' => '',
+	),
+	'feInterface' => array (
+		'fe_admin_fieldList' => '',
+	)
+);
+
+$TCA['tx_dlf_printer'] = array (
+	'ctrl' => array (
+		'title'     => 'LLL:EXT:dlf/locallang.xml:tx_dlf_printer',
+		'label'     => 'label',
+		'print' 	=> 'print',
+		'default_sortby' => 'ORDER BY label',
+		'delete'	=> 'deleted',
+		'dynamicConfigFile' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath($_EXTKEY).'tca.php',
+		'iconfile'	=> \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath($_EXTKEY).'res/icons/txdlflibraries.png',
```

Comment:
Please choose a different icon which better represents the purpose of this table.
The icon set can be found here: http://www.famfamfam.com/lab/icons/silk/